From 0d2fa5cd29b8ce60c5b1dfc2a68e09794e9073c8 Mon Sep 17 00:00:00 2001
From: Chris Leech <cleech@redhat.com>
Date: Fri, 30 Jan 2015 08:21:41 +0000
Subject: [PATCH] enabled test tool building for distcheck

set the --enable-debug flag when running distcheck, to test build the
debug tools as well

Signed-off-by: Chris Leech <cleech@redhat.com>
Signed-off-by: John Fastabend <john.r.fastabend@intel.com>
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index 69deda2..b1c381b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -139,3 +139,4 @@ dist_systemdsystemunit_DATA = lldpad.service lldpad.socket
 bashcompletiondir = $(sysconfdir)/bash_completion.d
 dist_bashcompletion_DATA = contrib/bash_completion/lldpad contrib/bash_completion/lldptool
 
+AM_DISTCHECK_CONFIGURE_FLAGS = --enable-debug
-- 
2.1.0

